scopes/harmony/graphql/graphql.ui.runtime.tsx

```diff
@@ -157,32 +154,3 @@ export class GraphqlUI {
 }
 
 GraphqlAspect.addRuntime(GraphqlUI);
-
```

Comment:
getIdFromObject is deprecated in apollo v3, and already handled by inMemoryCache